### DISCUSSION TITLE
Use texture as background?

### DISCUSSION THREAD

**Discussion by EnrichoUser** [bnpr/Malt#462] [Q&A] [answered]
Good day, is there a way to use an image as a background; something similar to an environment texture?

**Comment by pragma37** [comment, marked as answer]
Hi!

The render nodes have a Background screen pass by default:
![imagen](https://user-images.githubusercontent.com/36610999/209857170-dd2ac832-c05e-47a2-ba7d-c983da5eb719.png)

You can change the background color there to whatever you want, including an HDRI map:
![imagen](https://user-images.githubusercontent.com/36610999/209857386-6c12c2ab-436e-41db-a0ca-8671916625e8.png)

**Reply by EnrichoUser** [reply]
Thank you.
